server/services/prisonService.ts

```diff
@@ -1,57 +1,53 @@
 import { NotFound } from 'http-errors'
-import { Prison } from '../data/prisonRegisterApiTypes'
 import { HmppsAuthClient, PrisonRegisterApiClient, RestClientBuilder, VisitSchedulerApiClient } from '../data'
+import { Prison } from '../data/visitSchedulerApiTypes'
 
 const A_DAY_IN_MS = 24 * 60 * 60 * 1000
 
-export default class SupportedPrisonsService {
+export default class PrisonService {
```

Comment:
Renamed this as the 'supported' bit seemed redundant for this use case. So far, this service now just caches and returns list of all prison names (from Prison register) and handles getting the list of all prisons currently in VSiP